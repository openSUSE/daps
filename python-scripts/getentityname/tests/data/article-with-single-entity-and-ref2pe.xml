<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE article
[
  <!ENTITY % ent3 SYSTEM "entity-decl3.ent">
    %ent3;
]>

<article version="5.0" xml:lang="en"
  xmlns="http://docbook.org/ns/docbook">
  <title>Test Article</title>
  <para/>
</article>
