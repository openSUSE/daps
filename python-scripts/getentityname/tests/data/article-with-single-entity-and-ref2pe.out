entity-decl3.ent
